openff/toolkit/topology/molecule.py

```diff
@@ -6336,6 +6342,16 @@ def visualize(self, backend="rdkit", width=500, height=300):
                 import nglview as nv
             except ImportError:
                 raise MissingDependencyError("nglview")
+
+            if (
+                width is not None
+                or height is not None
+                or show_all_hydrogens is not None
+            ):
```

Comment:
```suggestion
            if (
                width is not None
                or height is not None
            ) or not show_all_hydrogens:
```
I think it makes sense to allow `mol.visualize(backend="nglview", show_all_hydrogens=True)`, even though it effectively does nothing.